tests/test_lmdeploy/test_model.py

```diff
@@ -585,6 +585,28 @@ def test_codellama_others():
     assert model is None
 
 
+def test_deepseek():
+    model = MODELS.get('deepseek')()
+    messages = [{
+        'role': 'system',
+        'content': 'you are a helpful assistant'
+    }, {
+        'role': 'user',
+        'content': 'who are you'
+    }, {
+        'role': 'assistant',
+        'content': 'I am an AI'
+    }, {
+        'role': 'user',
+        'content': 'hi'
+    }]
+    from transformers import AutoTokenizer
+    tokenizer = AutoTokenizer.from_pretrained('deepseek-ai/DeepSeek-V2-Lite', trust_remote_code=True)
```

Comment:
How about adding test cases for DeepSeek V3 and DeepSeek R1 here?

Reply:
Done in https://github.com/InternLM/lmdeploy/pull/3072